src/docs/modules/ROOT/pages/modeling-guidelines.adoc

```diff
@@ -493,14 +478,30 @@ Example:
 include::example$aspect-sample.ttl[tags=entity-declaration]
 ----
 
-When an Entity is defined with a Property which has no Characteristic, for example the shared
-`xref:entities.adoc#time-series-entity[Time Series]` Entity, the Entity needs to be refined in order
-to use it. This is done by declaring an Entity with `bamm:refines` and with a Property that in turn
-refines another Property. See the following example:
+==== Extending Entities
```

Comment:
This section must be extended (no pun intended): It must explain the uniqueness constraint for effective Property payload names across an Entity inheritance chain.

Reply:
In https://github.com/OpenManufacturingPlatform/sds-bamm-aspect-meta-model/issues/34#issuecomment-861163908 these two constraints are proposed:

    all Properties defined in an Entity hierarchy must have a unique payload name
    the combination of all Properties, incl. optional Properties, of an Entity hierarchy must be unique

These constraints is quite hard if the different entitites extending the abstract one live in different name spaces. Typically in one aspect model I would like to reuse the abstract entity but would not necessarily use all entities inheriting from this entity but only the ones in my own namespace?

Reply:
Yes, these are hard constraints. What you describe would (I assume) be addressed in Issue #36 - when resolving a model there might have to be some kind of an "import" mechanism that determines which models from which namespaces are taken into account. This means that the constraints apply only the resolved model (the closure of model elements referencing each other, even across namespaces), not necessarily globally to _all existing_ models.

Reply:
yes, I agree: this could tackle the problem and we can discuss it there. So all statements like "all Properties defined in an Entity hierarchy" using "all" or "an" only hold within this namespace-set (we need a name for it, you call the result a "resolved model") for the aspect model.